posts/2017-04-12-serverless-v1.11.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Separate Package and Deploy command, CloudWatch Log event support with Serverless v1.11
```

Comment:
We need catchier titles focusing on a main benefit.

Suggestion: "New Release: Trigger functions with cloudwatch events in Serverless v1.11"

cc @Stefmonge what do you think?